### PR TITLE
Add FAQ to docs

### PR DESCRIPTION
**Pull request by garethx**
## What kind of change does this PR introduce?

Adds an FAQ page to the docs
- [x] Documentation
- [x] Testing
- [ ] Ready to be merged - needs to be checked for accuracy.

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3662/commits/6c03ecd1aecdb58e066afb74040fc66b6866e881 is at https://pr3662.build.csb.dev/s/new.

**Comment by garethx** [comment]
Will discuss with @DannyRuchtie what to do about Screenshot 2020-03-09 at 16 46 41

File: packages/homepage/content/docs/1-faqs.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## Can I change the Node version used in a container sandbox?
+
+Container sandboxes run Node v10.18.1 by default. While this system version can't be changed, you can instead make use of the [node](https://www.npmjs.com/package/node) package. Add the version of the node package you would like to use to your sandbox in the same way you would any other package. Since npm runs scripts with the local `./node_modules/.bin` in the `PATH` ahead of the system copy, this means you can use this local version of node instead of the system version.
+
+## Can I open the terminal / console / test panel instead of the browser in a sandbox?
+
+Yes, the terminal, console, and problems tabs are all draggable. Click on the tab and drag it up into the bar alongside browser and tests. You can then re-order those items by dragging them in that bar. Whichever is 1st from left to right in the list of tabs is what opens first when other folks view the sandbox. The ordering is maintained within the sandbox. You can also achieve this change by setting a value for "view" in a [sandbox config file](/docs/configuration#sandbox-configuration). 
+
+## Are there any limitations with sandboxes?
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/1-faqs.md]:
```suggestion
## Are there any limitations with sandboxes?

```

**Comment by garethx** [comment]
Fixed up title length issues: Screenshot 2020-03-10 at 10 50 02

**Comment by CompuIves** [comment]
Nice!!

**Pull request merged**